### ISSUE TITLE
DaLI generated config overwrites account_methods section

### ISSUE THREAD

**Issue by orientalperil** [eprbell/rp2#136]
For the RP2 config file there is a section that lets you choose a different accounting method for each year:

https://github.com/eprbell/rp2/blob/aca7cf2b4cc702b3f802c3a00582b81dd886fef3/config/test_data_multi_method.ini#L6

If you put this in your config and then use DaLI to generate files again, it will be overwritten.  Also there is no way to express multiple methods for different years on the command line.

So are you supposed to generate files with DaLI and then manually put this into the config file every time?

**Comment by eprbell** [comment]
Good point. As of today the workaround is to edit manually the generated config, as you mentioned. The correct solution would be to add some pass-through sections in the DaLI config that would be just copied verbatim to the RP2 generated config. I'm currently working on adding support for per-wallet application to RP2 (see #135), so I don't have time to work on this now: if anybody wants to submit PRs, I'll review.
